### ISSUE TITLE
Incorrect error trace when including API as a type

### ISSUE THREAD

**Issue by sichvoge** [raml-org/raml-js-parser-2#518]
Lets assume I have the following example:

types.raml
```yaml
#%RAML 1.0
title: types def

types:
  song:
    type: object
    properties:
      name: string
    example:
      name: ss
```

api.raml
```yaml
#%RAML 1.0
title: testing include raml

types: !include types.raml
```

The parser does not throw any error when including the `types.raml` file. Why is that? Should `include` take the nodes and attach them to the node where its being used? Even changing `types` to `schemas` did not bring up any problems.

**Comment by ddenisenko** [comment]
That's what I get when parsing this file:

```
  "errors": [
    {
      "code": 11,
      "message": "Inheriting from unknown type",
      "path": "test48.raml",
      "range": {
        "start": {
          "line": 1,
          "column": 0,
          "position": 11
        },
        "end": {
          "line": 1,
          "column": 5,
          "position": 16
        }
      },
      "trace": [
        {
          "code": 11,
          "message": "Inheriting from unknown type",
          "path": "test49.raml",
          "range": {
            "start": {
              "line": 3,
              "column": 0,
              "position": 24
            },
            "end": {
              "line": 3,
              "column": 5,
              "position": 29
            }
          }
        }
      ]
    },
    {
      "code": 11,
      "message": "Specifying unknown facet: 'song'",
      "path": "test48.raml",
      "range": {
        "start": {
          "line": 4,
          "column": 2,
          "position": 38
        },
        "end": {
          "line": 4,
          "column": 6,
          "position": 42
        }
      },
      "trace": [
        {
          "code": 11,
          "message": "Specifying unknown facet: 'song' test48.raml",
          "path": "test48.raml",
          "range": {
            "start": {
              "line": 3,
              "column": 14,
              "position": 38
            },
            "end": {
              "line": 3,
              "column": 18,
              "position": 42
            }
          }
        }
      ]
    }
  ]
}
```

**Comment by carowright** [comment]
The second error :

```
{
      "code": 11,
      "message": "Specifying unknown facet: 'song'",
      "path": "test48.raml",
      "range": {
        "start": {
          "line": 4,
          "column": 2,
          "position": 38
        },
        "end": {
          "line": 4,
          "column": 6,
          "position": 42
        }
      },
      "trace": [
        {
          "code": 11,
          "message": "Specifying unknown facet: 'song' test48.raml",
          "path": "test48.raml",
          "range": {
            "start": {
              "line": 3,
              "column": 14,
              "position": 38
            },
            "end": {
              "line": 3,
              "column": 18,
              "position": 42
            }
          }
        }
      ]
}
```
has "path": "test48.raml" inside trace. I think it should be "path": "test49.raml" instead.

**Comment by ddenisenko** [comment]
Agree

**Comment by ddenisenko** [comment]
Changing ticket title to "Incorrect error trace when including API as a type" to avoid confusion.

**Comment by KonstantinSviridov** [comment]
The fix is going to be released in 1.1.6
